version.py

```diff
@@ -55,7 +57,7 @@ def process_version(
     # as we usually use git commit hashes
     build_metadata_length: int=12
```

Comment:
add new parameter; e.g. verbatim_version: str=None

add check: iff operation == set_verbatim, exactly this argument _must_ be set. the others (prerelease, build_metadata) _must not_ be set